TestCases/disc_adj_rans/cylinder/cylinder_Windowing.cfg

```diff
@@ -0,0 +1,416 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
As a general note on examples, I would try to clean the config as much as possible to really put the focus on the new options you added, for example if MG_LEVEL=0 all other MG options don't do anything, if you use the Roe scheme the JST coefficients are not important and so on.

Reply:
I have removed default options, that are unnecessary for a meaningful test case.